### PR TITLE
WRN-18693: Update testing library rules

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Update lint rules of testing

### Resolution
Update lint rules of testing

### Links
WRN-18693

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Review by MikyungKim — COMMENTED** [review]
🤔 Why eslint-plugin-testing-library module is not updated?

**Comment by hong6316** [comment]
> 🤔 Why eslint-plugin-testing-library module is not updated?

@MikyungKim 
That module will be updated on #45

File: index.js [hunk 1]
```diff
@@ -244,13 +244,38 @@ module.exports = {
...
+
 				// https://github.com/testing-library/eslint-plugin-testing-library
 				'testing-library/await-async-query': 'error',
 				'testing-library/await-async-utils': 'error',
-				'testing-library/no-await-sync-query': 'warn',
+				'testing-library/no-await-sync-query': 'error',
+				'testing-library/no-container': 'error',
+				'testing-library/no-debugging-utils': 'error',
 				'testing-library/no-dom-import': ['error', 'react'],
+				'testing-library/no-node-access': 'error',
```
Comment (MikyungKim) [inline index.js:266]: I think we can't add this rule since there is no way to test without node access right now. 🤔

Comment (hong6316) [reply index.js]: OK

File: index.js [hunk 2]
```diff
@@ -244,13 +244,38 @@ module.exports = {
...
+
 				// https://github.com/testing-library/eslint-plugin-testing-library
 				'testing-library/await-async-query': 'error',
 				'testing-library/await-async-utils': 'error',
-				'testing-library/no-await-sync-query': 'warn',
+				'testing-library/no-await-sync-query': 'error',
+				'testing-library/no-container': 'error',
+				'testing-library/no-debugging-utils': 'error',
 				'testing-library/no-dom-import': ['error', 'react'],
+				'testing-library/no-node-access': 'off',
```
Comment (MikyungKim) [inline index.js:266]: Can we just remove it? Should we "off" this?

File: strict.js [hunk 3]
```diff
@@ -43,13 +43,13 @@ module.exports = {
 		'no-array-constructor': 'error',
 		'no-cond-assign': ['error', 'except-parens'],
 		'no-console': 'warn',
-		'no-debugger': 'warn',
+		'no-debugger': 'error',
```
Comment (MikyungKim) [inline strict.js:46]: If these `warn` to `error` rules in the eslint:recommended, can we just remove them?

File: index.js [hunk 4]
```diff
@@ -235,7 +235,7 @@ module.exports = {
 				'**/?(*.)+(spec|test).[jt]s?(x)',
```
Comment (MikyungKim) [inline index.js:235]: Please update the glob pattern.

File: index.js (hunk 2 above)
Comment (hong6316) [reply index.js]: Either is fine, let's remove it.

File: index.js (hunk 4 above)
Comment (hong6316) [reply index.js]: OK

File: strict.js (hunk 3 above)
Comment (hong6316) [reply strict.js]: OK, let's remove it.
Not exactly, but before eslint 6.0.0(before May 2019), `eslint:recommend` already set 'no-debugger' to 'error'.
(https://github.com/eslint/eslint/commit/f42d0afd89874b459fce1eb1998247d53f9aa42b)

Comment (hong6316) [reply strict.js]: OK, let's remove it.
Not exactly, but before eslint 6.0.0(before May 2019), `eslint:recommend` already set 'no-debugger' to 'error'.
(https://github.com/eslint/eslint/commit/f42d0afd89874b459fce1eb1998247d53f9aa42b)

**Comment by MikyungKim** [comment]
Closing this in favor of #49 .

**Pull request closed without merging**